src/Npgsql/MultiplexingDataSource.cs

```diff
@@ -110,14 +110,14 @@ async Task MultiplexingWriteLoop()
 
                     connector = await OpenNewConnector(
                         command.InternalConnection!,
-                        new NpgsqlTimeout(TimeSpan.FromSeconds(Settings.Timeout)),
+                        TimeSpan.FromSeconds(Settings.Timeout),
```

Comment:
BTW if we're going into these micro-optimizations, you can also just pass Settings.Timeout here and not do TimeSpan.FromSeconds

Reply:
True. I think at that point I was thinking about passing `TimeSpan` since it's unambiguous, unlike `int`...

Reply:
It's a very internal/local context (it's not like we're exposing an API to a user here)... and if we think perf is important enough to remove new NpgsqlTimeout, well :)